src/Microsoft.OData.Core/ODataOutputContext.cs

```diff
@@ -489,7 +489,7 @@ internal virtual Task<ODataDeltaWriter> CreateODataDeltaWriterAsync(IEdmEntitySe
         /// The write must flush the output when it's finished (inside the last Write call).
         /// Since we don't want to support batch format extensibility (at least not yet) this method should remain internal.
         /// </remarks>
-        internal virtual ODataBatchWriter CreateODataBatchWriter(string batchBoundary)
+        internal virtual ODataBatchWriter  CreateODataBatchWriter(string batchBoundary)

```

Comment:
>   [](start = 41, length = 2)

why only add an extra space here? maybe revert this file from PR.

Reply:
file still showing up.

---
In reply to: [134626173](https://github.com/OData/odata.net/pull/951#discussion_r134626173) [](ancestors = 134626173)